src/js/components/SelectCard/SelectCard.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Icon } from '../Icon';
+import { getClassName } from '../../utils/helpers';
+
+export interface Props {
+  className?: string;
+  mods?: string;
+  iconName?: string;
+  text?: string;
+  iconStyle?: React.CSSProperties;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+}
+
+const SelectCard: React.FunctionComponent<Props> = ({
+  className,
+  mods,
+  iconName,
+  text,
+  iconStyle,
+  style,
+  children,
+  ...props
+}: Props) => {
+
+  const [selected, setSelected] = React.useState(false);
+
+  const handleClick = () => {
+    setSelected(!selected);
+  };
+
+  const cardClasses = getClassName(
+    className,
+    selected && `Card--selected`,
+    mods
+  );
+
+  return (
+    <div className={cardClasses} style={style} {...props} onClick={handleClick}>
```

Comment:
I wonder about writing this as an `input type=checkbox` with the checkbox hidden vs a `div` to support keyboard interaction natively and to use semantic HTML. Thoughts?